Mathlib/LinearAlgebra/Orientation.lean

```diff
@@ -115,14 +115,14 @@ theorem Orientation.reindex_symm (e : ι ≃ ι') :
 end Reindex
 
 /-- A module is canonically oriented with respect to an empty index type. -/
-instance (priority := 100) IsEmpty.oriented [Nontrivial R] [IsEmpty ι] : Module.Oriented R M ι where
```

Comment:
Can you explain in the PR description how you managed to eliminate these?

Reply:
I got an unused variable warning on these so removed them. Turns out they're redundant on master too. Added #8129 as a dependency PR.